Document target-scope setting in JSON API (#861)
digital-asset/docs.daml.com: docs/2.8.11/docs/json-api/index.rst

The HTTP JSON API Service does not validate the token but may need to decode it to extract information that can be used
to fill in request fields for party-specific request. How this happens depends partially on the token format you are using.

> If the Canton participant to which the JSON API is connected uses target-scopes, the configuration has to reflect that
> through the following setting:

Reviewer: ```suggestion
A Canton participant connected to the JSON API using target-scopes must be configured as follows:
```
